src/api/rpcspec/methods.ts

```diff
@@ -1279,13 +1279,73 @@ async fn main() {
 `,
   },
   // Submit a new class declaration transaction
-  /* {
+  {
     name: "starknet_addDeclareTransaction",
     params: {
       declare_transaction: BROADCASTED_DECLARE_TXN,
     },
     starknetJs: ``,
-  }, */
+    starknetGo: ``,
+    starknetRs: `use std::sync::Arc;
+
+use starknet::{
+  core::types::{
+    contract::SierraClass, FieldElement,
+    BroadcastedDeclareTransaction, BroadcastedInvokeTransactionV2, BroadcastedDeclareTransactionV3,
+    DataAvailabilityMode, ResourceBoundsMapping, ResourceBounds
+  },
+  macros::felt,
+  providers::{
+    jsonrpc::{HttpTransport, JsonRpcClient},
+    Provider, Url,
+  },
+};
+        
+#[tokio::main]
+async fn main() {
+  // Sierra class artifact. Output of the "starknet-compile" command
+  let contract_artifact: SierraClass =
+    serde_json::from_reader(std::fs::File::open("/path/to/contract/artifact.json").unwrap())
+    .unwrap();
+  // Class hash of the compiled CASM class from the "starknet-sierra-compile" command
+  let compiled_class_hash =
+    FieldElement::from_hex_be("COMPILED_CASM_CLASS_HASH_IN_HEX_HERE").unwrap();
```

Comment:
`compiled_class_hash` is already defined [here](https://github.com/NethermindEth/rpc-request-builder/pull/33/files#diff-3f7805a6ff96deb51896364a792288d8b8abde9ba78a70dbb451739fcc8f6768R1325)
```suggestion
```